### DISCUSSION TITLE
how to install when dependency gnome-icon-theme is not avilable?

### DISCUSSION THREAD

**Discussion by kaddkaka** [PintaProject/Pinta#589] [Q&A]
Hi, I tried to install pinta for the first time today on a laptop that is behind a proxy and I'm not allowed to add apt repositories.

The dependency on gnome-icon-theme seem to be the problem. What can I do? Can I lift/remove the icon-theme dependency?

```console
Ign:1 http://chuangtzu.ftp.acc.umu.se/ubuntu jammy/universe amd64 gnome-icon-theme all 3.12.0-4
Err:1 http://se.archive.ubuntu.com/ubuntu jammy/universe amd64 gnome-icon-theme all 3.12.0-4
  403  Forbidden [IP: ___]
E: Failed to fetch http://se.archive.ubuntu.com/ubuntu/pool/universe/g/gnome-icon-theme/gnome-icon-theme_3.12.0-4_all.deb  403  Forbidden [IP: ___]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

Is the icon theme available elsewhere?

**Comment by cameronwhite** [comment]
You might be okay to try removing the gnome-icon-theme dependency (I assume you're trying  to install a .deb package of Pinta?). If anything does go wrong it would most likely result in some standard GTK icons missing in the UI

**Reply by kaddkaka** [reply]
I have just tried to install it via apt, how do I do what you suggest?

**Reply by cameronwhite** [reply]
It would probably require editing the .deb package, which might not be the easiest. I'm not sure if your issue that's affecting gnome-icon-theme would also prevent you from downloading other dependences or pinta's .deb package itself too?

If you have permissions to install snap or flatpak packages, those might be easier avenues to try getting Pinta installed for your situation
